optimum/intel/commands/inc/quantize.py

```diff
@@ -0,0 +1,46 @@
+from argparse import REMAINDER
+
+
+def parse_args_inc_quantize(parser):
+    parser.add_argument("-o", "--opt", type=str, default="", help="optimization feature to enable")
+
+    parser.add_argument("-a", "--approach", type=str, default="auto", help="quantization approach (strategy)")
+
+    parser.add_argument("--config", type=str, default="", help="quantization configuration file path")
+
+    parser.add_argument(
+        "-b", "--bench", default=False, action="store_true", help="conduct auto_quant benchmark instead of enable"
+    )
+
+    parser.add_argument(
+        "-e", "--enable", default=False, action="store_true", help="only do enable, not overwrite or run program"
+    )
+
+    # positional
+    parser.add_argument(
+        "script",
+        type=str,
+        help="The full path to the script to be launched. " "followed by all the arguments for the script",
+    )
+
+    # script args
+    parser.add_argument("script_args", nargs=REMAINDER)
+
+
+from neural_coder.launcher import Launcher
+
+from ...neural_compressor.neural_coder_adaptor import NeuralCoderAdaptor
```

Comment:
Could this be moved to `optimum/intel/neural_compressor/.` (to have all the dependecy releated to INC there)

Reply:
I was designing completely based on your suggestion above - to mimic optimum's CLI, so that's why I created those files in those corresponding directory position. If you want our CLI to be `optimum-intel-cli inc quantize`, then those files have to be in those directories  (that's because `optimum-intel-cli` points to `optimum/intel/commands/`, so `optimum-intel-cli inc quantize` has to point to `optimum/intel/commands/inc/quantize')

Reply:
Added a suggestion in https://github.com/kaiyaointel/optimum-intel/pull/3